spec/models/mixins/assignment_mixin_spec.rb

```diff
@@ -18,11 +18,9 @@
       alert_set2.assign_to_tags([ct2], "vm")
       alert_set2.reload # reload ensures the tag is set
 
-      expect(test_class.assignments).to match_array(
-        [
-          {:assigned => alert_set, :assigned_to => "vm/tag/managed/environment/test"},
-          {:assigned => alert_set2, :assigned_to => "vm/tag/managed/environment/staging"},
-        ]
+      expect(test_class.assignments).to eq(
+        "vm/tag/managed/environment/test"    => [alert_set],
+        "vm/tag/managed/environment/staging" => [alert_set2],
       )
```

Comment:
Is the output of assignment actually used anywhere?  If so, then doesn't this break the callers?